tribuo-tutorials/README.md

```diff
@@ -0,0 +1,35 @@
+# Tribuo tutorials
+
+This project shows how to run [Tribuo Machine Learning](https://tribuo.org) algorithms with Conclave.
+
+The code is based on [Tribuo's tutorials](https://tribuo.org/learn/4.0/tutorials/), which cover the following examples:
```

Comment:
what code? Is this a sample or a tutorial? If a tutorial, shouldn't we say something like "The tutorials, which are based on...., cover the following example:"

Reply:
I've removed sample and stuck with tutorials. Let me know if I missed it anywhere.